### ISSUE TITLE
Running ./whitelist.sh locks pi-hole server hard locks computer

### ISSUE THREAD

**Issue by naota2k3** [anudeepND/whitelist#89]
I am running Pi-Hole on a Ubuntu Server 18-04.02 LTS server with 2 Gig of ram, 60 gigs hard drive, 2 logical hard drives in VMWARE ESX-i 6.5. When I attempt to run the whitelist script. It will attempt  rebuild the gravity list and the computer will completely lock up.   I am unable to get to it using console or ssh. I have to hard reboot the unit in order to gain access again.

**Comment by pahakalle** [comment]
Do you have large lists on pi-hole?
If you run `pihole -g` on the server will it lock up.

**Comment by naota2k3** [comment]
I have a stock install of Pi-hole on a ubuntu virtual machine. I wasn't running any custom white lists or black lists. I have noticed the that instance of pihole was crashing with pihole -up and pihole -g. I have reinstalled the operating system and made sure it was patched correctly. I was getting the same issue on 18-04 of ubuntu. I have reinstalled with 16-04 ubuntu and it seems to be working correctly. I think there is an issue with the way 18-04 is running causing it to lock u
